.gitignore

```diff
@@ -6,3 +6,7 @@ coverage
 log
 tmp
 .vagrant
+/node_modules
+/public
+/config
```

Comment:
typically the config directory is kind of important. Do you really want to ignore it?

Reply:
This is for jekyll and not rails, so I think it's fine.

Reply:
If there is a config directory i'm pretty sure it's kind of important. If there is no config directory, this addition shouldn't be necessary. Typically jekyll ist configured using a `_config.yml` in the root directory, so no config directory. I'm just wondering why this line was added. 

But this is definitely non-blocking.

Reply:
I think gitignore could be revised but in another pull request. There's also some content that was pushed and isn't necessary.

Reply:
> If there is a config directory i'm pretty sure it's kind of important. If there is no config directory, this addition shouldn't be necessary. Typically jekyll ist configured using a _config.yml in the root directory, so no config directory. I'm just wondering why this line was added.

This is me checking out from the regular master-based branches to gh-pages-based branches, so no real danger 😁